### PR TITLE
new audio video filetype colors

### PR DESCRIPTION
**Pull request by bigbadcapers**
- [x] New icons updated in Fabric CDN ([example](https://res-2-sdf.cdn.office.net/files/fabric-cdn-prod_20240805.001/assets/item-types/32/audio.svg))
- [x] Updated Fabric CDN url to new version.
- [x] yarn build/lage was run to update style-utilities .md
- [x] Tested within @fluentui/react-experiments

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/BaseButton-mount_base.html">650</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/BaseButton-mount.html">611</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Breadcrumb-mount_base.html">1704</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Breadcrumb-mount.html">1741</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Checkbox-mount_base.html">1715</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Checkbox-mount.html">1717</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/CheckboxBase-mount_base.html">1463</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/CheckboxBase-mount.html">1495</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/ChoiceGroup-mount_base.html">2933</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/ChoiceGroup-mount.html">2951</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/ComboBox-mount_base.html">678</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/ComboBox-mount.html">670</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/CommandBar-mount_base.html">6730</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/CommandBar-mount.html">6698</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/ContextualMenu-mount_base.html">13792</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/ContextualMenu-mount.html">14202</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/DefaultButton-mount_base.html">777</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/DefaultButton-mount.html">794</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/DetailsRow-mount_base.html">2217</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/DetailsRow-mount.html">2268</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/DetailsRowFast-mount_base.html">2199</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/DetailsRowFast-mount.html">2208</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/DetailsRowNoStyles-mount_base.html">2044</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/DetailsRowNoStyles-mount.html">2013</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Dialog-mount_base.html">2793</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Dialog-mount.html">2764</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/DocumentCardTitle-mount_base.html">235</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/DocumentCardTitle-mount.html">229</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Dropdown-mount_base.html">1990</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Dropdown-mount.html">1976</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/FocusTrapZone-mount_base.html">1178</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/FocusTrapZone-mount.html">1148</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/FocusZone-mount_base.html">1080</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/FocusZone-mount.html">1097</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/GroupedList-mount_base.html">42392</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/GroupedList-mount.html">42766</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/GroupedList-virtual-rerender_base.html">20542</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/GroupedList-virtual-rerender.html">20474</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">52645</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/GroupedList-virtual-rerender-with-unmount.html">52484</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/GroupedListV2-mount_base.html">239</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/GroupedListV2-mount.html">238</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/GroupedListV2-virtual-rerender_base.html">218</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/GroupedListV2-virtual-rerender.html">217</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">249</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">219</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/IconButton-mount_base.html">1127</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/IconButton-mount.html">1140</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Label-mount_base.html">351</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Label-mount.html">343</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Layer-mount_base.html">2785</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Layer-mount.html">2764</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Link-mount_base.html">390</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Link-mount.html">396</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/MenuButton-mount_base.html">988</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/MenuButton-mount.html">970</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/MessageBar-mount_base.html">21418</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/MessageBar-mount.html">21374</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Nav-mount_base.html">2050</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Nav-mount.html">2084</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/OverflowSet-mount_base.html">778</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/OverflowSet-mount.html">800</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Panel-mount_base.html">1798</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Panel-mount.html">1884</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Persona-mount_base.html">734</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Persona-mount.html">765</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Pivot-mount_base.html">920</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Pivot-mount.html">890</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/PrimaryButton-mount_base.html">922</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/PrimaryButton-mount.html">938</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Rating-mount_base.html">4703</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Rating-mount.html">4716</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/SearchBox-mount_base.html">958</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/SearchBox-mount.html">937</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Shimmer-mount_base.html">1916</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Shimmer-mount.html">1931</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Slider-mount_base.html">1334</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Slider-mount.html">1359</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/SpinButton-mount_base.html">2915</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/SpinButton-mount.html">2962</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Spinner-mount_base.html">395</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Spinner-mount.html">389</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/SplitButton-mount_base.html">1878</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/SplitButton-mount.html">1875</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Stack-mount_base.html">430</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Stack-mount.html">430</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/StackWithIntrinsicChildren-mount_base.html">868</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/StackWithIntrinsicChildren-mount.html">883</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/StackWithTextChildren-mount_base.html">2763</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/StackWithTextChildren-mount.html">2844</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/SwatchColorPicker-mount_base.html">6413</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/SwatchColorPicker-mount.html">6376</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/TagPicker-mount_base.html">1499</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/TagPicker-mount.html">1501</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Text-mount_base.html">403</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Text-mount.html">386</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/TextField-mount_base.html">969</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/TextField-mount.html">974</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/ThemeProvider-mount_base.html">856</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/ThemeProvider-mount.html">867</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/ThemeProvider-virtual-rerender_base.html">578</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/ThemeProvider-virtual-rerender.html">587</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1282</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1324</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Toggle-mount_base.html">637</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/Toggle-mount.html">628</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/buttonNative-mount_base.html">197</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32241/perf-test/buttonNative-mount.html">197</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: change/@fluentui-style-utilities-60215204-2c5b-469a-9aef-fc22d60bacc0.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-style-utilities-60215204-2c5b-469a-9aef-fc22d60bacc0.json:1]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentui-web-components-v3 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32241&commitId=76f07d92bc91a74c6cd53f51c3f29fc2671b18de&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar.Image.chromium.png|248| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

Comment (fabricteam) [inline change/@fluentui-style-utilities-60215204-2c5b-469a-9aef-fc22d60bacc0.json:1]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32241&commitId=76f07d92bc91a74c6cd53f51c3f29fc2671b18de&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.chromium.png|11| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Review by spmonahan — APPROVED** [review]

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: change/@fluentui-style-utilities-60215204-2c5b-469a-9aef-fc22d60bacc0.json (hunk 1 above)
Comment (bigbadcapers) [reply change/@fluentui-style-utilities-60215204-2c5b-469a-9aef-fc22d60bacc0.json:1]: resolved as noise, the PR does not change anything in the charting components, one of the reported regressions does not look different, the other has some pixels in the chart curve in the diff but does not look to be caused by the files touched in this PR

Comment (fabricteam) [inline change/@fluentui-style-utilities-60215204-2c5b-469a-9aef-fc22d60bacc0.json:1]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32241&commitId=29f34fe0cdeaf5cfc8d11452f3c7aae8daf8a43f&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.chromium.png|11| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Comment (bigbadcapers) [reply change/@fluentui-style-utilities-60215204-2c5b-469a-9aef-fc22d60bacc0.json:1]: **charting visual regressions:** resolved as noise, the PR does not change anything in the charting components, one of the reported regressions does not look different, the other has some pixels in the chart curve in the diff but does not look to be caused by the files touched in this PR

**persona visual regression:** the baseline actually had a broken image, and the new rev shows a profile pic. resolved that one as correct.

**Comment by bigbadcapers** [comment]
hi @microsoft/fui-wc could we request your review for this PR?

**Review by chrisdholt — APPROVED** [review]

**Comment by fabricteam** [comment]
## 🕵 fluentui-web-components-v3 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: change/@fluentui-style-utilities-60215204-2c5b-469a-9aef-fc22d60bacc0.json (hunk 1 above)
Comment (fabricteam) [inline change/@fluentui-style-utilities-60215204-2c5b-469a-9aef-fc22d60bacc0.json:1]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32241&commitId=8c69c0b2ef16f350422facebe1432c9bb0ffc16b&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Pull request merged**